randovania/games/dread/json_data/Cataris.txt

```diff
@@ -2673,6 +2680,7 @@ Extra - asset_id: collision_camera_031
       Any of the following:
           Spider Magnet or Space Jump
           Flash Shift and Grapple Beam
```

Comment:
Grapple Alone should suffice I think. Should be able to get above solid ground easily enough. And Space Jump can be changed to Use Spin Boost